app/src/main/java/org/kiwix/kiwixmobile/zim_manager/ZimManageViewModel.kt

```diff
@@ -89,12 +93,14 @@ class ZimManageViewModel @Inject constructor(
 ) : ViewModel() {
   sealed class FileSelectActions {
     data class RequestOpen(val bookOnDisk: BookOnDisk) : FileSelectActions()
```

Comment:
Shouldn't this be deleted? Or should it just be used instead of `NavigateTo`

I am not against some spring cleaning on this branch where you run the formatter over the codebase and delete any unused classes using the inspections